runtime/compiler/z/codegen/S390J9CallSnippet.hpp

```diff
@@ -133,6 +138,7 @@ class S390InterfaceCallSnippet : public TR::S390VirtualSnippet
 
    public:
 
+   // TODO: remove this ctor. thunk pointer is now mandatory
```

Comment:
Can we address this TODO in a new commit? We should do it now that we are in the area of this code.